src/Components/Organisms/FOSideBar/index.tsx

```diff
@@ -69,13 +106,87 @@ export const FOSideBar = memo((props: IFOSideBar) => {
         sx={sx}
         variant="persistent"
         anchor="left"
-        open={opens}
+        open={isDrawerOpen}
         {...materialUIProps}
       >
         <DrawerHeader>
-          <FAIconButton testID="icon-button-drawer" onClick={handleDrawerClose}>
+          <FAIconButton onClick={handleDrawerClose} testID="icon-button-drawer">
             {theme.direction === "ltr" ? (
-              <ChevronLeftIcon />
+              <Box
+                sx={{
+                  width: "100%",
+                  display: "flex",
+                  justifyContent: "flex-start",
+                }}
+              >
+                {logo ? (
+                  <>{logo}</>
+                ) : (
+                  <svg
```

Comment:
Separate this SVG code to new component e.g `@Atoms/Logo` here's the reference if you need it https://github.com/HZN-one/console/tree/development/src/Components/Atoms/Logo

Reply:
done